src/Pagination/index.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+import * as React from "react";
+
+import ButtonLink from "../ButtonLink";
+import Stack from "../Stack";
+import ChevronLeft from "../icons/ChevronLeft";
+import ChevronRight from "../icons/ChevronRight";
+import MAXIMUM_PAGES from "./consts";
+import Pages from "./components/Pages";
+import CompactPages from "./components/CompactPages";
+
+import type { Props } from "./index";
+
+const handlePageChange = (onPageChange, pageCount) => nextPageIndex => {
+  if (onPageChange && nextPageIndex <= pageCount && nextPageIndex >= 0) {
+    onPageChange(nextPageIndex);
+  }
+};
+
+const Pagination = (props: Props) => {
+  const {
+    pageCount,
+    selectedPage = 1,
+    previousLabel = "Previous",
+    nextLabel = "Next",
+    onPageChange,
+    dataTest,
+  } = props;
+  const pageChanged = handlePageChange(onPageChange, pageCount);
+
+  return (
+    <Stack direction="row" spacing="tight" align="center" dataTest={dataTest}>
```

Comment:
is not it row by default?

Reply:
Nope, because `Stack` is not flex block by default, you need to explicitly enable `flex` behavior with some prop.